connector/spanmetricsconnector/config.go

```diff
@@ -67,6 +67,10 @@ type Config struct {
 	// MetricsEmitInterval is the time period between when metrics are flushed or emitted to the configured MetricsExporter.
 	MetricsFlushInterval time.Duration `mapstructure:"metrics_flush_interval"`
 
+	// MetricsExpiration is the time period after which, if no new spans are received, metrics are considered stale and will no longer be exported.
+	// Default value (0) means that the metrics will never expire.
+	MetricsExpiration time.Duration `mapstructure:"metrics_expiration"`
```

Comment:
Should we add validation that this value is positive?

Reply:
Probably, if only for sanity sake. Currently, a negative number will skip an early return but **shouldn't** cause problems.

Reply:
I think feels like a potential stumbling block tho.  Let's add something to the validate function the enforces a non-negative duration.